docs/topology.md

```diff
@@ -0,0 +1,60 @@
+# Network Topologies in Kops
+
+Kops supports a number of pre defined network topologies. They are separated into commonly used scenarios, or topologies.
+
+Each of the supported topologies are listed below, with an example on how to deploy them.
+
+# AWS
+
+Kops supports the following topologies on AWS
+
+|      Topology     |   Value    | Description                                                                                                 |
+| ----------------- |----------- | ----------------------------------------------------------------------------------------------------------- |
+|   Public Cluster  |   public   | All masters/nodes will be launched in a **public subnet** in the VPC                                        |
```

Comment:
When this is merged it will go into both - we aren't calling out differences (yet) do we really need this in this PR? Maybe another PR for documenting the APIs?